packages/nx-plugin/src/executors/cli/executor.ts

```diff
@@ -15,44 +15,49 @@ export default async function runAutorunExecutor(
   terminalAndExecutorOptions: AutorunCommandExecutorOptions,
   context: ExecutorContext,
 ): Promise<ExecutorOutput> {
-  const { objectToCliArgs, formatCommandStatus } = await import(
-    '@code-pushup/utils'
-  );
+  const { objectToCliArgs, formatCommandStatus, logger, stringifyError } =
+    await import('@code-pushup/utils');
   const normalizedContext = normalizeContext(context);
   const cliArgumentObject = parseAutorunExecutorOptions(
     terminalAndExecutorOptions,
     normalizedContext,
   );
-  const {
-    dryRun,
-    verbose,
-    command: cliCommand,
-    bin,
-  } = terminalAndExecutorOptions;
+  const { command: cliCommand } = terminalAndExecutorOptions;
+  const { dryRun, verbose, bin, ...restArgs } = cliArgumentObject;
   const command = bin ? `node` : 'npx';
   const positionals = [
     bin ?? '@code-pushup/cli',
     ...(cliCommand ? [cliCommand] : []),
   ];
-  const args = [...positionals, ...objectToCliArgs(cliArgumentObject)];
+  const args = [...positionals, ...objectToCliArgs(restArgs)];
+  const executorEnvVariables = {
+    ...(verbose && { CP_VERBOSE: 'true' }),
+  };
   const commandString = formatCommandStatus([command, ...args].join(' '), {
     cwd: context.cwd,
+    env: executorEnvVariables,
   });
-  if (verbose) {
-    logger.info(`Run CLI executor ${command ?? ''}`);
-    logger.info(`Command: ${commandString}`);
-  }
+
   if (dryRun) {
     logger.warn(`DryRun execution of: ${commandString}`);
   } else {
     try {
+      logger.debug(`With env vars: ${executorEnvVariables}`);
```

Comment:
The way you're setting `CP_VERBOSE` will only apply to the inner command, but not logs from the executor, like the `logger.debug` statement above.

A simpler alternative would be to call `logger.setVerbose(verbose)`. This method sets `process.env['CP_VERBOSE']`, meaning the verbose setting will apply to the current process as well as child processes. Then you wouldn't need to pass the `env` option to the child process.

Reply:
THX